From c5952487b48cf5a31633a17973dbc89e391fda55 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Mon, 10 Feb 2025 17:10:13 +0100
Subject: [PATCH] curl: bump to ver. 8.11 for fixing error in updater

Bump curl to new version where error:
assert: (curl_multi_assign(downloader->cmulti, s, sdata)) == CURLM_OK
has been fixed for turris updater
---
 net/curl/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/curl/Makefile b/net/curl/Makefile
index 5210f8c8d..5c1fa5984 100644
--- a/net/curl/Makefile
+++ b/net/curl/Makefile
@@ -9,13 +9,13 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/nls.mk
 
 PKG_NAME:=curl
-PKG_VERSION:=8.10.1
+PKG_VERSION:=8.11.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://github.com/curl/curl/releases/download/curl-$(subst .,_,$(PKG_VERSION))/ \
 	https://curl.se/download/
-PKG_HASH:=73a4b0e99596a09fa5924a4fb7e4b995a85fda0d18a2c02ab9cf134bebce04ee
+PKG_HASH:=c7ca7db48b0909743eaef34250da02c19bc61d4f1dcedd6603f109409536ab56
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=COPYING
-- 
2.48.1

